0x05/주정윤/스택 수열.java

```diff
@@ -0,0 +1,54 @@
+import java.io.*;
+import java.util.*;
+
+public class Solution {
+    
+    
+	public static void main(String[] args) throws IOException {
+		
+		BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
+		BufferedWriter bw = new BufferedWriter(new OutputStreamWriter(System.out));
+		
+		StringBuilder sb = new StringBuilder();
+		
+		// N 입력  
+		int N = Integer.parseInt(br.readLine());
+		Stack<Integer> stack = new Stack<>();
+			
+		int temp =0; 
+		
+		for(int i=1; i<=N; i++) {
```

Comment:
같은 방법이지만. for 문에서 사용하는 i를 for문 내부에서 사용 안하는 것 같아서 저는 while에서 N을 하나씩 빼면서 반복문을 돌았어요! 
```suggestion
		while(N--> 0){
```